easybuild/easyconfigs/r/RDKit/RDKit-2021.03.4_fix-comic-neue-checksum.patch

```diff
@@ -0,0 +1,11 @@
+--- rdkit-Release_2021_03_4/Code/GraphMol/MolDraw2D/CMakeLists.txt.orig	2022-01-17 13:56:43.780726092 +0100
```

Comment:
@SIB-software A short description should be added on top of this patch, to clarify why this is needed
Author or upstream reference for this change should also be mentioned.

Note that this implies changing the SHA256 checksum for this patch file too...